src/components/pages/membership_form/subpages/AddressPage.vue

```diff
@@ -186,14 +184,22 @@ const membershipApplication = computed( (): MembershipApplication => {
 
 const isDirectDebitPayment = computed( (): boolean => store.state.membership_fee.values.type === 'BEZ' );
 
-const addressSummary = computed( (): MembershipAddress => {
+const addressSummary = computed( (): Address => {
```

Comment:
Here, I used `Address` instead of `MembershipAddress` because I wanted to reuse `src/components/pages/donation_form/DonorSummarySection.vue` component. The component prints the address. (yes, it maybe needs to be renamed)

Or should I keep using `MembershipAddress` and create a new component to print address?